src/test/java/test/testng2321/TestMultipleInstance.java

```diff
@@ -0,0 +1,50 @@
+package test.testng2321;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+
+public class TestMultipleInstance {
```

Comment:
Instead of having this as a separate class, can you please have this moved into `test.thread.parallelization.issue1773.IssueTest` as a test method ? This test class can be invoked by test method in this class and checked for exit status.